contrib/certbot/files/tmpfiles.conf

```diff
@@ -0,0 +1,3 @@
+d /etc/letsencrypt
+d /var/lib/letsencrypt
+d /var/log/letsencrypt 0700
```

Comment:
shouldn't certbot also have its own user and whatnot

Reply:
though maybe not, idk

Reply:
the tmpfiles.conf is from arch linux which doesn't have a certbot user, neither does alpine.  I think in normal use it is creating certificates for things run as lots of different users.

Reply:
sure, makes sense